pkg/util/unstructured.go

```diff
@@ -32,8 +32,8 @@ func ToUnstructuredList(obj runtime.Object) (*unstructured.UnstructuredList, err
 		if err != nil {
 			return nil, err
 		}
-		for _, obji := range items {
-			ud, err := toUnstructured(obji)
+		for _, item := range items {
+			ud, err := toUnstructured(item)
```

Comment:
This is a more opinionated change. I understand there's already an `obj` in scope, but there's also an `items` and normally while iterating over a list of `foos` the thing one gets is a `foo`, so `item` seems like a pretty reasonable choice.

Reply:
I'm fine with `item`. Thank you.